<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetFilterWidget</class>
 <widget class="QWidget" name="SetFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="filterNumL">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sourceDirTB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icegui.qrc">
          <normaloff>:/dvg/icegui/images/open.png</normaloff>:/dvg/icegui/images/open.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="sourceDirLE"/>
        </item>
        <item>
         <widget class="QLabel" name="extensionsL">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="filterPB">
        <property name="text">
         <string>Фильтр ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterNameLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="destinationDirTB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icegui.qrc">
          <normaloff>:/dvg/icegui/images/open.png</normaloff>:/dvg/icegui/images/open.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destinationDirLE"/>
      </item>
      <item>
       <widget class="QSpinBox" name="paramSB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="useChB">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icegui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sourceDirTB</sender>
   <signal>pressed()</signal>
   <receiver>SetFilterWidget</receiver>
   <slot>sourceDirTB_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destinationDirTB</sender>
   <signal>pressed()</signal>
   <receiver>SetFilterWidget</receiver>
   <slot>destinationDirTB_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterPB</sender>
   <signal>pressed()</signal>
   <receiver>SetFilterWidget</receiver>
   <slot>filterPB_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useChB</sender>
   <signal>toggled(bool)</signal>
   <receiver>SetFilterWidget</receiver>
   <slot>useChB_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sourceDirTB_pressed()</slot>
  <slot>destinationDirTB_pressed()</slot>
  <slot>filterPB_pressed()</slot>
  <slot>useChB_toggled(bool)</slot>
 </slots>
</ui>
